jax/_src/numpy/lax_numpy.py

```diff
@@ -2813,6 +2813,17 @@ def tril_indices_from(arr: ArrayLike, k: int = 0) -> tuple[Array, Array]:
   return tril_indices(arr_shape[-2], k=k, m=arr_shape[-1])
 
 
+@util._wraps(np.fill_diagonal)
```

Comment:
We should document the inplace parameter. You could do it similarly to `put()`: https://github.com/google/jax/blob/06306274e50d12ddaeab963bdad951084b44638b/jax/_src/numpy/lax_numpy.py#L5165-L5175